examples/Time-Series/time_series_keithley2600.py

```diff
@@ -0,0 +1,146 @@
+    linefreq = float(smu.ask("print(localnode.linefreq)"))
+
+# %% Save the data columns in a pandas DataFrame and inspect 
+data = pd.DataFrame({
+    'Time (s)': times - times[0],
+    'Current (A)': currents
+})
+
+# For 1 MOhm and 10V bias we expect ~ 0.01 mA -> in my case it measured 0.009 mA but this should be tolerable. If your measurement is totally off check all the connections
+print(data.tail())
+print("Compare the difference of duration = {}s and the end time of {}s in our DataFrame".format(duration, data["Time (s)"][-1:].values))
+print("The timing is not accurate since there is a delaye from the for-loop. If you do only care about the number of datapoints this is not a problem")
+
+# %% Advanced example with buffer
+# At the time of writing the low level function to use the buffer of the device is not available via attributes
+# So we use the write() and read() methods to access them like the beeper above
+
+# It turns out, that the USB-serial RS-232 connection is too slow to transfer data from the buffer back to the pc for longer or faster measurements
+# Therefore we increase the baud rate. The factory-selected baud rate is 9600. Possible values = [300, 600, 1200, 2400, 4800, 9600, 19200, 38400, 57600, 115200].
+
+visa_address = list_resources()[0] # automatically receive device address
+
+
+
+smu = Keithley2600(visa_address) # connect to the instrument with the standard baud_rate
+smu.write("serial.baud = {}".format(115200)) # Set the baud rate at the instrument to the fastest possible
+
+# Connect again and tell also python to use the new baud_rate
+with Keithley2600(visa_address, baud_rate=115200) as smu:
+
+    # First define input parameters
+    smu.reset()
+    smu.ChA.wires_mode = "2"
+
+    # Prepare buffer
+    smu.write("smua.nvbuffer1.clear()")  # smua corresponds to smu.ChA
+    #smu.write("smua.measure.interval = t/5")   # Time between measurements in (s); ! If to small, smallest possible steps will be chosen 
+    smu.write("smua.nvbuffer1.appendmode = 1")
+    smu.write("smua.nvbuffer1.collecttimestamps	= 1") # True
+    smu.write("smua.measure.count = {}".format(n_datapoints))
+
+
+    smu.ChA.source_voltage = voltage
+    sleep(0.1) # wait here to give the instrument time to react
+    smu.ChA.measure_current(nplc=nplc, current=i_range, auto_range=False) # with auto_range=True it takes very long and sleep(X) might to be increased
+    sleep(0.1)
+
+    # Apply bias
+    smu.ChA.apply_voltage(compliance_current=i_compliance)  # Somehow I would expect this to turn source on but it does not
+    sleep(0.1)
+    smu.ChA.source_output = "ON" # needed to turn source on
+
+    # Measure current
+    smu.write("smua.measure.i(smua.nvbuffer1)")
+    #smu.write("waitcomplete()") # somehow not needed
+    sleep(duration + 1) # Wait the time the measurement takes
+
+    currents = np.fromstring(smu.ask("printbuffer(1, {}, smua.nvbuffer1)".format(n_datapoints)), sep=",")
```

Comment:
`smu.values("printbuffer(1, {}, smua.nvbuffer1)".format(n_datapoints))` should do the same thing (only gives plain lists, not numpy arrays).